clients/dos/src/main/kotlin/DosModel.kt

```diff
@@ -90,8 +99,8 @@ data class JobRequestBody(
     /** The key of the previously uploaded ZIP file to scan. */
```

Comment:
out of scope: In a separate commit / PR, this could be renamed to `ScanJobRequestBody`.